language/move-prover/bytecode-to-boogie/src/spec_translator.rs

```diff
@@ -524,7 +524,13 @@ impl<'a> SpecTranslator<'a> {
             }
         }
         self.error(
-            &format!("cannot resolve struct `{}`", id),
+            // Specifications can use types with no corresponding StructDefinitionIndex in the
```

Comment:
Do you mean StructHandleIndex? StructDefinitionIndex is only for those structs defined in the module.